check_process

```diff
@@ -25,3 +25,7 @@
 ;;; Options
 Email=
 Notification=none
+;;; Upgrade options
+	; commit=2cc84310aeff7055342b445c1aee01d4183d5ae2
+		name=MAJ 11.6.3
+		manifest_arg=domain=DOMAIN&path=PATH&admin=USER&use_web_account=WEB_ACCOUNT&is_public=1&
```

Comment:
`use_web_account=WEB_ACCOUNT` that's for sure won't work.

Reply:
Got it, `DOMAIN` and `PATH` are specific values, which aren't define by the check_process. Well path could be.
But those value are defined by package_check during the tests, depending on whether it's a test on sub path or root  and during change_url as well.
Anyway, it works for those 2 values, but it won't work for WEB_ACCOUNT as it's not a value package_check will read.

Reply:
Well, I have to reread the f****** manual

Reply:
Oh the fucking m***** says also:
> In case of lack of name, the commit will be used.
And so on in case of lack of manifest arguments, the default arguments of the check process will be used.

I should have reread it as well ;)
Should be tried. Maybe you did already though.

But still I can't have a working test to try to debug it !

Reply:
> Should be tried. Maybe you did already though.

In fact, it's working like that since the beginning...

I will close this PR then!